solver/src/solver/http_solver.rs

```diff
@@ -129,195 +130,14 @@ impl HttpSolver {
         }
     }
 
-    fn map_tokens_for_solver(&self, orders: &[LimitOrder], liquidity: &[Liquidity]) -> Vec<H160> {
-        let order_tokens = orders
-            .iter()
-            .flat_map(|order| [order.sell_token, order.buy_token]);
-        let liquidity_tokens = liquidity.iter().flat_map(|liquidity| match liquidity {
-            Liquidity::ConstantProduct(amm) => amm.tokens.into_iter().collect::<Vec<_>>(),
-            Liquidity::BalancerWeighted(amm) => amm.reserves.keys().copied().collect(),
-            Liquidity::BalancerStable(amm) => amm.reserves.keys().copied().collect(),
-        });
-
-        order_tokens
-            .chain(liquidity_tokens)
-            .collect::<HashSet<_>>()
-            .into_iter()
-            .collect()
-    }
-
-    fn token_models(
-        &self,
-        token_infos: &HashMap<H160, TokenInfo>,
-        price_estimates: &HashMap<H160, f64>,
-        buffers: &HashMap<H160, U256>,
-    ) -> HashMap<H160, TokenInfoModel> {
-        token_infos
-            .iter()
-            .map(|(address, token_info)| {
-                let external_price = match price_estimates.get(address).copied() {
-                    Some(price) if price.is_finite() => Some(price),
-                    _ => None,
-                };
-                (
-                    *address,
-                    TokenInfoModel {
-                        decimals: token_info.decimals,
-                        external_price,
-                        normalize_priority: Some(if &self.native_token == address { 1 } else { 0 }),
-                        internal_buffer: buffers.get(address).copied(),
-                    },
-                )
-            })
-            .collect()
-    }
-
-    fn map_orders_for_solver(&self, orders: Vec<LimitOrder>) -> HashMap<usize, LimitOrder> {
-        orders.into_iter().enumerate().collect()
-    }
-
-    fn order_models(
```

Comment:
Moved :point_down: outside of `impl HttpSolver` block to make it easier to unit test.